src/components/Forms/ImageUpload/index.js

```diff
@@ -7,7 +7,7 @@ import { useUploadImage } from '../../../hooks/images';
 import AvatarImage from '../../AvatarImage';
 import { CTAButton } from '../../Layout/CTAButton';
 
-export default ({ userData, userId, onUploadDone }) => {
+export default ({ userData, userId, onUploadDone, showLabel }) => {
   const [uploadImageState, uploadImage] = useUploadImage();
 
   useEffect(() => {
```

Comment:
Um die Endlosschleife des Renderns zu verhindern, könntest du erstmal onUploadDone aus dem Dependency Array entfernen. Ist nicht ideal, machen wir aber überall im Code so, um das zu verhindern, da wir am Anfang noch nicht wussten, wie man das gescheit handelt. Mir machen da mal eine größere Refactoring Aktion draus. Eigentlich handelt man das mit useCallback: https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
Oh, hier zeigt es einen komischen Ausschnitt aus dem Code an. Ich meine eigentlich weiter unten im useEffect in Zeile 13.

Reply:
Okay, probier ich auch mal aus. Hatte aber leider so oder so an der Stelle nicht geklappt, dass sich das Bild dann in userData updated. Trotzdem sehr gut zu wissen.